src/test/java/net/snowflake/ingest/streaming/internal/datatypes/IcebergStructuredIT.java

```diff
@@ -464,4 +471,25 @@ private void assertMap(String dataType, Map<?, ?> value) throws Exception {
     JsonNode expectedNode = value == null ? null : objectMapper.valueToTree(value);
     assertThat(actualNode).isEqualTo(expectedNode);
   }
+
+  private void removeNullFields(JsonNode node) {
```

Comment:
add comment on why this method is needed?

Also we should have null value comparison too no?

Reply:
This method is used to compare {} and {key: null}. As this two values are the same for iceberg struct.

Reply:
I see. we pass in key=null, but we get back missing-key?

Reply:
Yes, both are the same.